internal/pkg/cli/flag.go

```diff
@@ -97,6 +97,10 @@ const (
 
 	taskIDFlag    = "task-id"
 	containerFlag = "container"
+
+	valuesFlag        = "values"
+	overwriteFlag     = "overwrite"
+	inputFilePathFlag = "cli-input-yaml"
```

Comment:
I may have missed previous discussion on this; apologies if that's the case. What do you think about just `input-yaml`, without the `cli-`?

Reply:
This is because[ aws cli](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/iotevents/create-input.html) uses `--cli-input-yaml` as their flag name for a similarly-purposed flag